src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -1063,10 +1063,6 @@ String getNameForCandidate(String nameOrAlias) {
     return candidateAliasesToNameMap.get(nameOrAlias);
   }
 
-  String getCodeForCandidate(String nameOrAlias) {
```

Comment:
In [this comment](https://github.com/BrightSpots/rcv/pull/639#discussion_r1141238943), you mentioned possibly reverting / getting rid of some lines in `TabulatorSession`. Is that no longer feasible?

Reply:
I pushed a commit showing the effect of that. Depends on what we want -- we can get rid of `getNameForCandidate` in `ResultsWriter`, which generates results that use the candidate code instead of the candidate name. See the diff [here](https://github.com/BrightSpots/rcv/pull/690/commits/3f121b7a69aefc25d749838e64d82244458b07dc).

I'm not sure how the `_expected.csv`s are used, so I don't know which format is more useful, but it seems likely that it's more useful with canonical names (i.e. reverting #3f121b7, the next commit) and keeping the code as it was.

Let me know which you prefer.

Reply:
The `_expected.csv`s are just used for our tests.

Hard for me to answer this question... might be best for @tarheel or @chughes297 to weigh in? It appears that with the above commit you linked, it leaves our tests as-is, which is probably preferred?

Reply:
Are the outputted CSVs never used by election administrators? If so, then I suppose it doesn't matter.

Reply:
The real output CSV are definitely used by admins. @HEdingfield is just referring specifically to the `_expected.csv` files that are in the test directories.

Apologies if I missed a relevant part of the discussion -- I'm just looking at this comment thread -- but I think ideally the output CSV would have the canonical names in one column and then perhaps a separate column that lists any alias that are used for each candidate (separated by a delimiter).

Reply:
Hm-- let's separate adding both canonical names and aliases for another task?

For now, I think the question is: should the output CSVs have the candidate code or the canonical name?

The current PR and the existing codebase use the candidate code.
Reverting the last commit in this PR will use the canonical name, and simplify `ResultsWriter` a smidge.

Reply:
We should use canonical names wherever possible, I don't believe it's very useful for jurisdictions to get results using candidate codes.

Reply:
Great. I think that makes sense too. I reverted the last commit, and this PR should be ready to go now.

Reply:
> Hm-- let's separate adding both canonical names and aliases for another task?

@chughes297 @tarheel do y'all think it's worth creating a new issue for this? If so, is it something we'd need to include in 1.4.0?

Reply:
Yes, I think it would be useful to list aliases in the output spreadsheet. I doubt it's a requirement for v1.4, but @chughes297 can comment.

Reply:
Created #705 for this.